### PR TITLE
Created interactive list feature

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
The following changes has been added in this branch:

1. it will contain methods related to the status updates (completed: true / false).
2. Add event listener to the checkbox.
3. Update items object's value for completed key upon user actions.
4. Implement a function for the "Clear all completed" button (use filter() method).
5. Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -1,141 +1,111 @@
...
-        });
-      });
+        // add data, checkbox and delete button to li
+        li.innerHTML = `<input type = "checkbox" class="checkbox" ${checked}> ${item.description} <button class="edit"><i class="fa fa-edit" aria-hidden="true"></i></button> <button class="delete"><i class="fa fa-trash" aria-hidden="true"></i></button>`;
 
-      const checkmarked = document.querySelectorAll('.checkbox');
-      checkmarked.forEach((btn, index) => {
-        btn.addEventListener('click', () => this.completedCheck(index));
+        todoListItems.appendChild(li);
       });
```
Comment (roshan-bajgain) [inline src/utils.js]: - [ ] It has an event listener on the checkbox (change). So kindly implement it

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @mirwaisfarahi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Your app is working properly and has well-structured code. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/utils.js (hunk 1 above)
Comment (mirwaisfarahi) [reply src/utils.js]: This line of code is not there with this branch. The completedCheck() method is implemented in Status class and called in utils.js File.

**Review by vic778 — APPROVED** [review]
Hi @Mirwais-Farahi      , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Pull request merged**